src/components/Home/Home.css

```diff
@@ -0,0 +1,42 @@
+.profile {
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+    margin: 50px 20px 0px;
+    padding-bottom: 50px;
+    text-align: start;
```

Comment:
I don't think there is such a value for text-align ... also you don't need to use `text-align:left` since it is the default behaviour

Reply:
if i don't use it the text will be in the center, which i don't want!

Reply:
`text-align : left then `